package.json

```diff
@@ -1010,6 +1019,12 @@
         "key": "L",
         "when": "notebookEditorFocused && !inputFocus && !notebookOutputInputFocused",
         "command": "runme.keybinding.noop"
+      },
+      {
+        "command": "runme.aiGenerate",
+        "key": "win+;",
+        "mac": "cmd+;",
+        "when": "resourceExtname==.md || resourceExtname==.markdown"
```

Comment:
`notebookType == runme` might be a better choice here instead of keying on file extension